ansible_inventory_server/maasrest.py

```diff
@@ -85,7 +106,7 @@ class MaasInventoryHandler(MaasRequestHandler):
     async def create_response(self, session):
         result = defaultdict(lambda: [])
         for m in await self.get_machines(session):
-            ip_addresses = filter_ip_addresses(m['ip_addresses'], self.json)
```

Comment:
m['interface_set'] would be enough because this is the only part of m you are using